fades/main.py

```diff
@@ -260,6 +260,9 @@ def go(argv):
     # run forest run!!
     python_exe = 'ipython' if args.ipython else 'python'
     python_exe = os.path.join(venv_data['env_bin_path'], python_exe)
+    # add the virtualenv /bin path to the child PATH.
```

Comment:
please leave an empty line above the comment so we have a "block of code" (it's the style of the rest of the text)